plugins/org.jkiss.dbeaver.ext.oceanbase/src/org/jkiss/dbeaver/ext/oceanbase/model/plan/OceanbasePlanJSON.java

```diff
@@ -29,28 +29,25 @@
     public OceanbasePlanJSON(JDBCSession session, String query) throws DBCException {
         this.dataSource = (OceanbaseMySQLDataSource) session.getDataSource();
         this.query = query;
-        try (JDBCPreparedStatement dbStat = session.prepareStatement(getPlanQueryString())) {
-            try (JDBCResultSet dbResult = dbStat.executeQuery()) {
-                List<OceanbasePlanNodeJSON> nodes = new ArrayList<>();
+        try{
+            JDBCPreparedStatement dbStat = session.prepareStatement(getPlanQueryString());
+            JDBCResultSet dbResult = dbStat.executeQuery();
```

Comment:
The point of `try-with-resources` here was to close result set and query after exiting their scopes.

Please bring it back or call `.close()` on both `dbResult` and `dbStat` in `finally` block.